test/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+IF(ArborX_TEST_MEMORY)
+  FIND_PROGRAM(MEMORYCHECK_COMMAND NAMES valgrind)
+  IF(MEMORYCOMMAND MATCHES "-NOTFOUND")
+    MESSAGE(FATAL_ERROR
+	    "Memory checking in tests enabled but could not find the valgrind "
+	    "executable. Please install valgrind.")
+  ENDIF()
+  SET(CTEST_MEMORYCHECK_COMMAND "${MEMORYCHECK_COMMAND}")
```

Comment:
Are you sure this variable is needed at all if we do `ctest -T MemCheck`?

Reply:
I thought so, but it seems to run without it as well now.